envoy/http/codec.h

```diff
@@ -568,6 +568,12 @@ class ClientConnection : public virtual Connection {
    * @return RequestEncoder& supplies the encoder to write the request into.
    */
   virtual RequestEncoder& newStream(ResponseDecoder& response_decoder) PURE;
+
+  /**
+   * True if the codec should await for the request encoding completion in addition
+   * to the response decoding completion before the stream can be closed.
```

Comment:
I think this is a bit confusing due to encode/decode meaning separate things for streams vs codecs.

maybe qualify a bit more:?

True if the codec should await for the request end stream to be encoded to the codec in addition to the response end stream being decoded from the codec before the stream can be closed.

Reply:
Ack, will update in the nest commit (resolving quic sanitizer issue).